examples/userdemo/pgf_fonts.py

```diff
@@ -5,7 +5,9 @@
 
 """
 
+import matplotlib as mpl
 import matplotlib.pyplot as plt
+mpl.use("pgf")
```

Comment:
Not allowed to do this before importing pyplot... Should one change the other examples as well?

Reply:
That shouldn't be a problem, switching to a noninteractive backend is always allowed.